pkg/machinery/api/cluster/cluster.pb.go

```diff
@@ -8,18 +8,16 @@ package cluster
 
 import (
 	context "context"
-	reflect "reflect"
-	sync "sync"
-
 	proto "github.com/golang/protobuf/proto"
 	duration "github.com/golang/protobuf/ptypes/duration"
+	common "github.com/talos-systems/talos/pkg/machinery/api/common"
 	grpc "google.golang.org/grpc"
 	codes "google.golang.org/grpc/codes"
 	status "google.golang.org/grpc/status"
 	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
 	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
-
-	common "github.com/talos-systems/talos/pkg/machinery/api/common"
+	reflect "reflect"
```

Comment:
looks like it's not formatted to our best practices here (talos imports grouped together as the last block)

Reply:
yeah, that was my bad. Should be fixed now.